frameworks/CSharp/aspnetcore/PlatformBenchmarks/BenchmarkApplication.cs

```diff
@@ -14,27 +14,27 @@ namespace PlatformBenchmarks;
 
 public partial class BenchmarkApplication
 {
-    private readonly static AsciiString _applicationName = "Kestrel Platform-Level Application";
-    public static AsciiString ApplicationName => _applicationName;
-
-    private readonly static AsciiString _crlf = "\r\n";
-    private readonly static AsciiString _eoh = "\r\n\r\n"; // End Of Headers
-    private readonly static AsciiString _http11OK = "HTTP/1.1 200 OK\r\n";
-    private readonly static AsciiString _http11NotFound = "HTTP/1.1 404 Not Found\r\n";
-    private readonly static AsciiString _headerServer = "Server: K";
-    private readonly static AsciiString _headerContentLength = "Content-Length: ";
-    private readonly static AsciiString _headerContentLengthZero = "Content-Length: 0";
-    private readonly static AsciiString _headerContentTypeText = "Content-Type: text/plain";
-    private readonly static AsciiString _headerContentTypeJson = "Content-Type: application/json";
-    private readonly static AsciiString _headerContentTypeHtml = "Content-Type: text/html; charset=UTF-8";
-
-    private readonly static AsciiString _dbPreamble =
-        _http11OK +
-        _headerServer + _crlf +
-        _headerContentTypeJson + _crlf +
-        _headerContentLength;
-
-    private readonly static AsciiString _plainTextBody = "Hello, World!";
+    private static ReadOnlySpan<byte> _applicationName => "Kestrel Platform-Level Application"u8;
+    public static ReadOnlySpan<byte> ApplicationName => _applicationName;
```

Comment:
Why the indirection here?

```suggestion
    public static ReadOnlySpan<byte> ApplicationName => "Kestrel Platform-Level Application"u8;
```